scripts/inc.installer.sh

```diff
@@ -1331,7 +1343,7 @@ get_prop() {
   #check known .prop files using get_file_prop
   local propfile propval
   for propfile in $PROPFILES; do
-    test "$propval" && break 2 || propval="$(get_file_prop $propfile $1 2>/dev/null)"
```

Comment:
you sure we don't need to break the loop here twice on purpose?

Reply:
We don't, break 2 is what has been keeping the device_name for loop from working correctly lately.